parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java

```diff
@@ -349,12 +369,34 @@ public Builder withColumnIndexTruncateLength(int length) {
     }
 
     /**
-     * Set Bloom filter info for columns.
+     * Set max Bloom filter bytes for related columns.
+     *
+     * @param maxBloomFilterBytes the max bytes of a Bloom filter bitset for a column.
+     * @return this builder for method chaining
+     */
+    public Builder withMaxBloomFilterBytes(int maxBloomFilterBytes) {
+      this.maxBloomFilterBytes = maxBloomFilterBytes;
+      return this;
+    }
+
+    /**
+     * Set Bloom filter column names.
+     *
+     * @param columns the columns which has bloom filter enabled.
+     * @return this builder for method chaining
+     */
+    public Builder withBloomFilterColumnNames(Set<String> columns) {
+      this.bloomFilterColumns = columns;
+      return this;
+    }
+
+    /**
+     * Set expected columns distinct number for Bloom filter.
      *
      * @param columnExpectedNDVs the columns expected number of distinct values in a row group
      * @return this builder for method chaining
      */
-    public Builder withBloomFilterInfo(Map<String, Long> columnExpectedNDVs) {
+    public Builder withBloomFilterColumnNdvs(Map<String, Long> columnExpectedNDVs) {
```

Comment:
Is this already released? Otherwise, we're breaking the API here.

Reply:
Not release yet.